src/main/java/com/kodilla/ecommercee/User.java

```diff
@@ -0,0 +1,98 @@
+package com.kodilla.ecommercee;
+
+import com.kodilla.ecommercee.temp.Cart;
+import com.kodilla.ecommercee.temp.Order;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "users")
+public class User {
+    private long id;
```

Comment:
należy dodać @Id, @GeneratedValue(strategy = GenerationType.AUTO), @NotNull, @Column(name = "user_id") i podobnie przy kolejnych kolumnach

Reply:
Tyle, że ja użyłem tu adnotacji przy akcesorach.
Możemy ustalić, że wrzucamy przy polach, ale zrobiłem jak mnie uczyli :)

Reply:
Od początku dodajemy "@Getter" i "@Setter", więc chyba nie będziemy tworzyć akcesorów. Kod jest wówczas prostszy i przyjemniejszy w odbiorze :)

Reply:
a to są już akurat dwie szkoły ;)
dla mnie jedno i drugie spoko, ale więcej plusów miałem na akcesory. Tak czy inaczej jeśli ustalimy jedną linię, że przy polach, to niechaj będą pola, chociaż znając życie pewnie będę bronił racji będąc w mniejszości :P